src/helpers/constants/ios.ts

```diff
@@ -1,6 +1,6 @@
 export const LOCAL_PATH_TO_CIO_NSE_FILES = `./node_modules/customerio-expo-plugin/src/helpers/native-files/ios`;
 export const IOS_DEPLOYMENT_TARGET = '13.0';
-export const CIO_SDK_VERSION = '~> 2.0.0';
+export const CIO_SDK_VERSION = "'>= 2.0.0', '< 3.0'";
```

Comment:
any particular reason we moved away from the previous syntax? because I think `~> 2.0.0` means the same thing.

Reply:
Customers complained of the following error when we use `~> 2.0.0`
```
Command `pod install` failed.
└─ Cause: CocoaPods could not find compatible versions for pod "CustomerIOTracking":
  In Podfile:
    CustomerIO/MessagingPushAPN (~> 2.0.0) was resolved to 2.0.0, which depends on
      CustomerIOMessagingPushAPN (= 2.0.0) was resolved to 2.0.0, which depends on
        CustomerIOMessagingPush (= 2.0.0) was resolved to 2.0.0, which depends on
          CustomerIOTracking (= 2.0.0)

    customerio-reactnative (from `../node_modules/customerio-reactnative`) was resolved to 2.2.0, which depends on
      CustomerIOMessagingInApp (~> 2.1.0-beta.2) was resolved to 2.1.0, which depends on
        CustomerIOTracking (= 2.1.0)

    customerio-reactnative (from `../node_modules/customerio-reactnative`) was resolved to 2.2.0, which depends on
      CustomerIOTracking (~> 2.1.0-beta.2)

```

Reply:
Wondering the same as Shahroz. Rest, the PR looks good to me.

Reply:
```suggestion
export const CIO_SDK_VERSION = '~> 2.0';
```

I think that this syntax is the same as `>= 2.0.0', '< 3.0`. 

I think this is the case based off of [the docs](https://guides.cocoapods.org/syntax/podfile.html#pod) for `~>` behavior. 

I did try `'~> 2.0'` in a cocoapods project and it did install 2.1 of the iOS SDK.

Reply:
Using `'~> 2.0'` also works